etl/steps/data/garden/un/2025-03-12/migrant_stock.meta.yml

```diff
@@ -0,0 +1,113 @@
+# NOTE: To learn more about the fields, hover over their names.
+definitions:
+  common:
+    presentation:
+      topic_tags:
+        - Migration
+      grapher_config:
+        originUrl: "https://ourworldindata.org/migration"
+    processing_level: minor #minor for most indicators, major for data that has been manipulated (e.g. 5 year change, share of child migrants)
+    unit: ""
+    short_unit: ""
+    display:
+      numDecimalPlaces: 0 # 0 for most indicators, 1 for percentages/ shares
+  description_un:
+    migrant_stock: International migrant stock at mid-year by age and sex and by region, country or area of destination, 1990-2024
+    share_migrant_stock: International migrant stock as percentage of the total population by age and sex and by region, country or area of destination, 1990-2024
+    migrants_per_age: International migrant stock at mid-year by age and sex and by region, country or area of destination, 1990-2024
+    share_age_migrants: Percentage distribution of the international migrant stock by age and sex and by region, country or area of destination, 1990-2024
+
+dataset:
+  update_period_days: 730 # had biyearly releases in the past, maybe they are going back to it?
+
+
+tables:
+  migrant_stock:
+    variables:
+      # Immigrant stock
+      immigrants_all:
+        title: Total number of international immigrants
+        description_short: People living in a given country who were born in another country.
+        description_from_producer: "{definitions.description_un.migrant_stock}"
+      immigrants_female:
+        title: Total number of international immigrants - female
+        description_short: The total number of female people living in a given country who were born in another country.
+        description_from_producer: "{definitions.description_un.migrant_stock}"
+      immigrants_male:
+        title: Total number of international immigrants - male
+        description_short: The total number of male people living in a given country who were born in another country.
+        description_from_producer: "{definitions.description_un.migrant_stock}"
+      # Immigrant share
+      immigrant_share_of_dest_population_all:
+        title: Share of the population that was born in another country
+        unit: "%"
+        short_unit: "%"
+        description_short: Immigrants as a share of the total population.
+        description_from_producer: "{definitions.description_un.share_migrant_stock}"
+        display:
+          numDecimalPlaces: 1
+      immigrant_share_of_dest_population_female:
+        title: Share of the population that was born in another country - female
+        unit: "%"
+        short_unit: "%"
+        description_short: Female immigrants as a share of the total (female) population.
```

Comment:
I like the clarification in the brackets! I always wonder what exactly the denominator is and now I no longer have to :)